server/src/internalClusterTest/java/org/elasticsearch/repositories/RepositoriesServiceIT.java

```diff
@@ -81,5 +82,11 @@ public void testUpdateRepository() {
 
         final Repository updatedRepository = repositoriesService.repository(repositoryName);
         assertThat(updatedRepository, updated ? not(sameInstance(originalRepository)) : sameInstance(originalRepository));
+
+        // check that a noop update does not verify.
+        internalCluster().startDataOnlyNode(Settings.builder().put(Environment.PATH_REPO_SETTING.getKey(), createTempDir()).build());
+        assertAcked(
```

Comment:
The new node does not share the same `path.repo` as the rest of the cluster and therefore fails verification. The PUT would fail if it did any verification, thus checking that the verification does not run.

I added a bit more to the comment to explain this.

Reply:
Ah thanks, sorry for being slow there :) Sweet idea for a test :)